Secrets/KeysExample.swift

```diff
@@ -4,3 +4,9 @@ struct Keys {
         static let clientKey = ""
     }
 }
+
+func swiftLintTest() {
+    let someForceCast = NSNumber() as! Int
+    let colonOnWrongSide :Int = 0
```

Comment:
Colon Violation (Low Severity): When specifying a type, always associate the colon with the identifier